iosApp/iosAppTests/Pages/NearbyTransit/NearbyTransitViewTests.swift

```diff
@@ -170,22 +170,31 @@ final class NearbyTransitViewTests: XCTestCase {
     struct LoadedStops {
         let predictionStops: [String]
         let scheduleStops: [String]
+        let pinnedRoutes: [String]
     }
 
     func setUpSut(
         _ objects: ObjectCollectionBuilder,
         _ loadPublisher: PassthroughSubject<LoadedStops, Never>,
-        now: Date? = nil
+        now: Date? = nil,
+        _ pinnedRoutes: Set<String> = []
     ) -> NearbyTransitView {
         let nearbyVM = NearbyViewModel()
         nearbyVM.nearbyState = getNearbyState(objects: objects)
         nearbyVM.alerts = .init(objects: objects)
 
         let predictionPub = PassthroughSubject<[String], Never>()
         let schedulePub = PassthroughSubject<[String], Never>()
-        Publishers.Zip(predictionPub, schedulePub).sink { predictionStops, scheduleStops in
+        let pinnedRoutesPub = PassthroughSubject<[String], Never>()
+
+        let pinnedRoutesRepository = MockPinnedRoutesRepository(
+            initialPinnedRoutes: pinnedRoutes,
+            onGet: { pinnedRoutesPub.send(pinnedRoutes.sorted()) }
+        )
+
+        Publishers.Zip3(predictionPub, schedulePub, pinnedRoutesPub).sink { predictionStops, scheduleStops, _ in
             loadPublisher.send(
-                LoadedStops(predictionStops: predictionStops, scheduleStops: scheduleStops)
+                LoadedStops(predictionStops: predictionStops, scheduleStops: scheduleStops, pinnedRoutes: [])
```

Comment:
Why include `pinnedRoutes` in `LoadedStops` if we aren’t going to put its actual value there?